open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/resources/application.properties

```diff
@@ -59,7 +59,8 @@ authentication.mode=token
 # component visibility for different roles
 # these properties are used to create a model for UI to show or hide components
 # role.visibleComponents.ROLE_NAME=coma,separate,values
-role.visibleComponents.USER=asset-catalog,about
+role.visibleComponents.USER=asset-catalog,about,asset-lineage,vertical-lineage
+role.visibleComponents.ADMIN=all
```

Comment:
i believe "all" is unnecessary, we should stick with @bogdan-sava's approach when user doesn't have anything configured it will default to provide access to everything

Reply:
I feel like this approach is more "mistake proof". In case someone forgets to configure the permissions for a role it should not get access to everything, the permissions for accessing everything should be given knowingly. 
I have discussed this this approach with @bogdan-sava and we agreed that this is a better handling of the "admin" role.

Reply:
@marius-patrascu the user should initially have a ROLE attached, such as "ADMIN" or "USER", so this means that "all" is still irrelevant if the role includes it or not